cmd/iq_test.go

```diff
@@ -35,18 +36,46 @@ import (
 )
 
 func TestIqApplicationFlagMissing(t *testing.T) {
-	output, err := executeCommand(rootCmd, "iq")
+	output, err := executeCommand(rootCmd, iqCmd.Use)
 	assert.Contains(t, output, "Error: required flag(s) \""+flagNameIqApplication+"\" not set")
 	assert.NotNil(t, err)
 	assert.Contains(t, err.Error(), "required flag(s) \""+flagNameIqApplication+"\" not set")
 }
 
 func TestIqHelp(t *testing.T) {
-	output, err := executeCommand(rootCmd, "iq", "--help")
+	output, err := executeCommand(rootCmd, iqCmd.Use, "--help")
 	assert.Contains(t, output, "go list -json -m all | nancy iq --"+flagNameIqApplication+" your_public_application_id --"+flagNameIqServerUrl+" ")
 	assert.Nil(t, err)
 }
 
+func TestIqCommandPathInvalidName(t *testing.T) {
+	origConfig := configOssi
+	defer func() {
+		configOssi = origConfig
+	}()
+	// not sure why calling executeCommand() fails as part of test suite, but is fine individually.
+	//_, err := executeCommand(rootCmd, iqCmd.Use, "--path", "invalidPath", "-a", "appId")
+	configOssi = types.Configuration{Path: "invalidPath"}
+	err := doIQ(iqCmd, []string{})
+
+	assert.Error(t, err)
+	assert.Contains(t, err.Error(), fmt.Sprintf("invalid path value. must point to '%s' file. path: ", GopkgLockFilename))
+}
+
+func TestIqCommandPathInvalidFile(t *testing.T) {
+	origConfig := configOssi
+	defer func() {
+		configOssi = origConfig
+	}()
+	// not sure why calling executeCommand() fails as part of test suite, but is fine individually.
+	//_, err := executeCommand(rootCmd, iqCmd.Use, "--path", GopkgLockFilename, "-a", "appId")
```

Comment:
So these are a fix it later thing??

Reply:
> So these are a fix it later thing??

Yes, or at least an "understand it better later" thing. Added TODO comments. cbcbbe3